config_templates/gretel/navigator_ft/navigator-ft-time-series.yml

```diff
@@ -0,0 +1,11 @@
+schema_version: "1.0"
+name: "navigator-ft-time-series" 
+models:
+  - navigator_ft:
+      data_source: __tmp__
+      group_training_examples_by: null # Set to column name of your choice
+      order_training_examples_by: null # Set to column name of your choice
```

Comment:
Is this the right way of exposing this param that's dependent on the customer provided dataset?

Reply:
Our other blueprint configs can be used directly without user input (this isn't completely true on all cases, though), so this new in the sense that they always have to provide input.